### DISCUSSION TITLE
Bausatz OpenDTU HW Erfahrungen

### DISCUSSION THREAD

**Discussion by hrpv** [tbnobody/OpenDTU#959] [Q&A] [answered]
Hat jemand Erfahrungen mit diesem Bausatz?
https://wechselrichter-versand.de/openDTU-S-fuer-Hoymiles-Serie-HMT-HMS-Bausatz/HY10012.3
Die löten wohl den CMT2300A bereits auf
Ich habe meinen CMT2300A ohne Platine verdrahtet, es funktioniert alles, aber nochmal will ich mir das Gefummel nicht antun.

**Comment by ms49434** [comment, marked as answer]
Der Bausatz ist in Ordnung, habe mir selber einen gekauft mit Display.

**Reply by hrpv** [reply]
o.k. danke für die Info

**Reply by Andy-Op** [reply]
Bevor man den ESP lötet sollte man ihn erst flashen. Ich hatte einen Defekten erwischt. Wurde anstandslos getauscht. Lieferant ist i.O..

**Comment by mermayer** [comment]
Dieser Bausatz wurde schon in der OpenDTU/Ahoy Facebook Gruppe besprochen und von einigen als potenziell Lebensgefährlich eingestuft, wenn man das interne Netzteil nutzt. Grund: Kein Schutzleiter und dieser wäre dringend notwendig, weil die Antenne außen einen Metallring besitzt, der über das Antennenkabel mit der Masse (GND) der Platine verbunden ist.

**Reply by Andy-Op** [reply]
Lebensgefährlich wird's nur, wenn man meint 230 Volt heranführen zu müssen. Ich hab ein altes iPhone Netzteil mit dem USB-C Kabel als Speisung. Da ist gar nix gefährlich. Wenn man auf die Idee kommt zu löten sollte man auch minimale Kenntnisse von elektrischen Gefahren haben. Unter 60 Volt DC passiert gar nichts, solange man nicht mit der Zunge kurzschließt...

**Reply by Marc9485** [reply]
Lebensgefährlich??? Oh Mann... 
Wie sollen denn bitte 230V bei einem Fehler / Kurzschluss auf dem Antennenstecker anliegen??? Da ist ein Trafo eingebaut. Galvanische Trennung nennt man das

**Reply by Andy-Op** [reply]
Hast den Bausatz überhaupt schon mal gesehen? Da gibt's ein uninsoliertes 230 Volt Netzteil was die Niederspannung erzeugt. Da kommt man primär sehr leicht mit den dicken Fingern dagegen. Da nützt dir galvanische Trennung auch nix...

**Reply by Marc9485** [reply]
Man merkt Sie haben keine Ahnung von Elektrik... Ja, ich habe mir den Bausatz angeschaut. Und es IST ein isoliertes Netzteil aufgrund des eingebauten Trafos. Wenn jemand mit seinen dicken Wurstfingern an den Trafo kommt passiert nichts! Wer mit seinen Fingern da dran kommt der kommt ja auch nicht zufällig da dran sondern weil er ja daran arbeitet. Und die 1. Regel der 5 Sicherheitsregeln ist "Freischalten"

**Reply by Andy-Op** [reply]
Ich hab den Bausatz gerader vor mir liegen. Wo soll die 230 Volt vom Netzkabel isoliert sein? Ach darüber wollen Sie nicht reden. Lassen se es lieber...

**Reply by Marc9485** [reply]
Ja, ich rede nicht mit einem Fachmann/Kollegen von daher macht das keinen Sinn zu diskutieren...
Vielleicht einfach mal nach Schutzklassen googeln... 
Oder sich z.B. mal nen Hifi-Verstärker (Metall-Gehäuse) oder aktiv Subwoofer von JBL oder Magnat (Metall Rückplatte) anschauen aber "Class AB" (analog Verstärker). Diese Geräte werden ebenfalls mit 230V betrieben, haben nur einen Eurostecker (2-polig L & N) und somit keinen PE am Gehäuse anliegen da dieses als Masse genutzt wird. Und diese Geräte dürfen alle auch in Deutschland betrieben und verkauft werden. Und ich sags nochmal einfach mal "galvanische Trennung" und "Schutzklassen" googeln bevor weiter wild rumgepoltert wird.

**Reply by Andy-Op** [reply]
Schutzklassen? Welche Schutzklasse haben wir denn bei 230 Volt? Ja es geht um den aufgelöteten Sperrwandler und der hat neben einer Sekundär- auch eine Primärseite. Das man an dem Kunstgehäuse keine Schlag bekommen ist jedem klar. Kunststoff ist ein Isolator. Den braucht man auch nicht zu erden. Leider liegen die 230 Volt unten an der Platine blank. Das heißt man brückt mit dem Finger L&N. Lernt man nebenbei im ERSTEN Ausbildungsjahr des Elektrohandwerks! Und wenn man den Bausatz selbst zusammen baut, gibt es halt keinen Hersteller der dort für einen zusätzliche Berührungsschutz sorgt. Entsprechend deiner verworren Äußerungen empfehle ich dir nichts zu zerlegen, wo Berührungsspannungen größer 60 Volt sein könnten.

**Comment by Rustimation** [comment]
Nachdem ich mit dem Verlöten des CMT2300A Chips beim Blinkyparts Bausatz grandios gescheitert bin, habe ich mir den oben angesprochenen Bausatz bei Wechselrichter-versand.de gekauft. Aus den oben erwähnten Gründen ohne eingebautes Netzteil (sicher gut gemeint aber etwas flimsy) und auch ohne Display, da ich das Teil sowieso hinter einem Dachbalken verstecken musste. Ein Gehäuse drum rum und alles funktioniert wie gewünscht. 
Hat man keine oder wenig Erfahrung beim Löten von winzigen SMD Bauteilen rate ich eher dazu, sich einen Bausatz mit vormontierem CMT Sender zu kaufen - auch wenn das Blinkly Teil etwas günstiger daherkommt und auch ein sehr schönes 3D gedrucktes Gehäuse hat.

**Reply by Rustimation** [reply]
Inzwischen gibt es auch den hier auf den Github Seiten von openDTU angesprochenen Bausatz bei Blinkyparts mit eine vorgelöteten CMT Sendermodul. Den Aufpreis für die Handarbeit ist es allemal wert.

**Comment by Omega13x** [comment]


**Reply by Andy-Op** [reply]
Dann gib mal einen Link an, wo man so eine Platine herbekommt.

**Reply by Andy-Op** [reply]
Die müssen einem als Bausatz dann schon gefallen. Der von wechselrichter-versand ist schön aufgeräumt und perfekt für DIY. Wüsste auch nicht, was man bei der 2x Modul-Lösung komplizierter machen muss.
